src/manager.cpp

```diff
@@ -67,7 +67,22 @@ Manager::Manager(
         iFace->register_method(
             "CollectFRUVPD",
             [this](const sdbusplus::message::object_path& i_dbusObjPath) {
-            this->collectSingleFruVpd(i_dbusObjPath);
+            if (m_worker.get() != nullptr)
+            {
+                nlohmann::json l_sysCfgJsonObj = m_worker->getSysCfgJsonObj();
+                if (dbusUtility::isHostRunning() &&
+                    jsonUtility::isVPDRecollectableAtRuntime(l_sysCfgJsonObj,
```

Comment:
With this check here, I have a confusion related to the behavior of the API.
When called over Dbus it will work only when host is running but when called internally will work irrespective of host state.
What should be updated on Doxygen then?